build.sbt

```diff
@@ -84,8 +84,10 @@ lazy val prometheusClientMetrics = libraryProject("prometheus-client-metrics")
     description := "Support for Prometheus Metrics on the client",
     libraryDependencies += prometheusClient
   )
-  .dependsOn(client % "compile;test->test")
-
+  .dependsOn(
+    client % "compile;test->test",
+    clientMetrics % "compile"
```

Comment:
Does this project depend on anything other than core now?  I don't think it does.  And then maybe the package name is `org.http4s.metrics.prometheus`.  Then it's a non-breaking change when the server implementation is made common.

Reply:
Sure. I'll do that.